core/mobile-manager/src/backend/iOSHost.ts

```diff
@@ -17,14 +56,15 @@ export class IOSHost extends MobileHost {
    * Start the backend of an IOS app.
    */
   public static async startup(opt?: { mobileHost: { device: MobileDevice }, ipcHost?: IpcHostOptions, iModelHost?: IModelHostConfiguration }): Promise<void> {
-    // enable once IOSDevice is working
-    // if (!opt) {
-    //   opt = { mobileHost: { device: new IOSDevice() } };
-    // }
-
-    // if (!opt.ipcHost) {
-    //   opt.ipcHost = { socket: new IpcWebSocketBackend() };
-    // }
+    if (!opt) {
+      opt = { mobileHost: { device: new IOSDevice() } };
+    }

```

Comment:
i think this should be something like::

```ts
  public static async startup(opt?: { iModelHost?: IModelHostConfiguration }): Promise<void> {
    const device = new IOSDevice();
    const socket = new IpcWebSocketBackend();
    (global as any).MobileDeviceRpcImpl = device; // for native side
    return MobileHost.startup({ ...opt, mobileHost: { device }, ipcHost: { socket } });
  }
```

Reply:
I was trying to allow you to pass in a custom socket or device...teams sometimes need a quick patch or workaround...

Reply:
but the way you have it you can't supply iModelHost options without supplying a device. Maybe:
```ts
  public static async startup(opt?: { mobileHost?: { device: MobileDevice }, ipcHost?: IpcHostOptions, iModelHost?: IModelHostConfiguration }): Promise<void> {
    const device = opt?.mobileHost?.device ?? new IOSDevice();
    const socket = opt?.ipcHost?.socket ?? new IpcWebSocketBackend();
    (global as any).MobileDeviceRpcImpl = device; // for native side
    return MobileHost.startup( {...opt, mobileHost: { device }, ipcHost: { socket } });
  }
```

Reply:
That's better. I copied the startup function signature over from the base class...didn't think of just changing it :-)